.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# FTL files will be reviewed by the l10n team
+*.ftl @firefox-devtools/performance-l10n
```

Comment:
I created this team and added @flodolo there, with write access to this repository (this seems to be necessary according to github's doc), and maintainer role for this team so that he can also add more people if needed later.

Reply:
I think it would be good to limit this review to only include English ftl files since it's our main file that needs to be reviewed. We wouldn't want to disturb :flod all the time we pull the ftl files from Pontoon :)
It could be like this: `locales/en-US/*.ftl`.

It's also what Firefox Monitor does: https://github.com/mozilla/blurts-server/blob/7c3b3ff0f7b0565e75cf80086eabaf62d82a3068/docs/CODEOWNERS#L2
(With the exception that we use `en-US` as our main English folder. I wonder if we should also rename it to `en` only?) Probably @flodolo knows what the best option is.

Reply:
Good idea to only target en-US, since you will need to merge from the branch to main.

As for the locale code, I'd keep en-US, since potentially you could also have en-CA and en-GB.

Reply:
Makes sense, I'll update this :-)  Thanks!